docs/source/models.rst

```diff
@@ -1,44 +1,59 @@
-Models
-======
+Architectures
```

Comment:
the file should be renamed architectures.rst

Reply:
done in 
9eb7885